packages/foam-vscode/src/features/commands/open-resource.ts

```diff
@@ -20,7 +20,7 @@ const feature: FoamFeature = {
     context.subscriptions.push(
       vscode.commands.registerCommand(
         OPEN_COMMAND.command,
-        async (params: { uri: URI }) => {
+        async (params: { uri: URI; chooseTemplate: boolean }) => {
```

Comment:
A `chooseTemplate` param in the `OPEN_COMMAND` doesn't feel right to me.
I know that I put too much responsibility in this command, that's the reason why I introduced the `CREATE_NOTE` command

Reply:
Yeah, logically it doesn't make a lot of sense - I was just hacking to get something to work. I can move this to the `CREATE_NOTE` command - I guess that makes more sense